source/more-info/backup-emergency-kit.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title: "Backup emergency kit"
+description: "Information about backup emergency kit"
+---
+
+The backup emergency kit contains the encryption key needed to recover your backup.
+Backups are encrypted (AES-128) and you need this information to decrypt the backup when you want to restore your system from that backup.
+
+## Storing the backup emergency kit somewhere safe
+
+1. To download the emergency kit, go to {% my backup title="**Settings** > **System** > **Backups**" %}
+2. If it is your first time defining backup settings, select **Setup automatic backup** and download the encryption key.
+   - You can also download the key again later under configuration.
+
+    ![Screenshot showing the encryption key in the download dialog for the backup emergency kit](/images/more-info/backup_emergency_kit_01.png)
+
+3. Store the kit somewhere safe, outside of the Home Assistant system.
+   - Without the encryption key, there is no way to restore the backup.
+
+## Generating a new encryption key
+
+The first time when you set up your backups, an encryption key is generated. But you can generate new ones. Old backups will still have the generated encryption key, but new backups will use the encryption key you defined yourself.
```

Comment:
Reading this, "generating" may not be the correct term to summarize this paragraph. The UI speaks about changing and to me it's more about replacing your encryption key. 

The paragraph can be rephrased differently:
* since you cannot generate (multiple) new ones, but there is only one encryption key in place at a certain time. 
* old refers here to the currently already existing backup's, while new refers to backups that will be created after you replaced your encryption key. 
* there's no option to define something yourself, the encryption key is generated and presented to you.